### PR TITLE
Minimum PHP version in `composer.json`

### PR DESCRIPTION
**Pull request by maximal**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
+        "php": ">=7.1.0",
```
Comment (rob006) [inline composer.json]:
```suggestion
        "php": "^7.2",
```

Comment (samdark) [reply composer.json]: 7.2 since we'll need event dispatcher.

**Comment by maximal** [comment]
Okay. I just took it from app template.

File: composer.json (hunk 1 above)
Comment (rob006) [reply composer.json]: So it official that Yii 3 will require PHP 7.2?

Comment (machour) [reply composer.json]: 7.1 security supports ends on Dec 1st 2019, I guess it makes sense

Comment (samdark) [reply composer.json]: @rob006, yes.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**